e2e-tests/cypress/e2e/journeys/insurance/accessibility-statement-signed-in.spec.js

```diff
@@ -0,0 +1,34 @@
+import partials from '../../partials';
+import { PAGES } from '../../../../content-strings';
+import { ROUTES } from '../../../../constants';
+
+const CONTENT_STRINGS = PAGES.ACCESSIBILITY_STATEMENT_PAGE;
+
+const { ROOT, ALL_SECTIONS, ACCESSIBILITY_STATEMENT } = ROUTES.INSURANCE;
+
+context('Accessibility statement page - Insurance - Signed in', () => {
+  let dashboardUrl;
+
+  beforeEach(() => {
+    cy.completeSignInAndGoToApplication().then((refNumber) => {
+      dashboardUrl = `${Cypress.config('baseUrl')}${ROOT}/${refNumber}${ALL_SECTIONS}`;
+    });
+
+    partials.footer.supportLinks.accessibilityStatement().click();
+
+    cy.assertUrl(`${Cypress.config('baseUrl')}${ACCESSIBILITY_STATEMENT}`);
```

Comment:
Not a blocker but i've started using a const like this at the top of the page to make URLs easy to read:

```js
const baseUrl = Cypress.config('baseUrl');
```